cli/src/semgrep/dependency_aware_rule.py

```diff
@@ -163,6 +168,7 @@ def generate_reachable_sca_findings(
                 )
                 if lockfile_path is None:
                     continue
+                scanned_lockfiles.append(lockfile_path)
```

Comment:
wont this be all lockfiles with a finding not necessarily scanned lockfiles? aka it will miss lockfiles without findings

Reply:
note: that's ok since if the PR did not introduce a new reachable finding, the lockfile doesnt need to be scanned so it's ok to miss it